lib/aha-services/version.rb

```diff
@@ -1,3 +1,3 @@
 module AhaServices
-  VERSION = "1.23.12"
+  VERSION = "1.23.15"
```

Comment:
There were steps along the way on this branch. This was mainly for deploying to staging purposes.